CondFormats/L1TObjects/interface/L1MuPacking.h

```diff
@@ -82,6 +82,11 @@ class L1MuUnsignedPackingGeneric : public L1MuPacking {
           << "L1MuUnignedPacking::packedFromIdx: warning value " << idx << "exceeds " << nbits << "-bit range !!!";
     return (unsigned)idx;
   };
+
+private:
+  int signFromPacked(unsigned packed) const = 0;
```

Comment:
Also here: why not to make another virtual method in the base class with the second `unsigned int` parameter, and avoid these useless final methods in the derived classes?